### ISSUE TITLE
Got an unknown version error, can you add?

### ISSUE THREAD

**Issue by bmwkk** [Floris497/mac-pixel-clock-patch-V2#55]
---- BEGINNING MD5 HASH SUMS ---- version: 10.12 build:16A270f

     otool CoreDisplay: 2dafa191cffb24a26e077aacf222b45a
 otool CoreDisplay.bak: NO FILE (this is okay)
           CoreDisplay: 0ab7c1c835cd3905f0673110309aad50
       CoreDisplay.bak: NO FILE (this is okay)

This is the xxx patcher.command md5 result. Can you add the code? Thanks

**Comment by alexanderzinolidis** [comment]
+1

**Comment by Floris497** [comment]
Yes, could you try to run "patch.command patch v1" see if it works :)

**Comment by alexanderzinolidis** [comment]
Patching NVDAGK100Hal with patch version 1

The code is still the same.. Patch did seem to run, but was probbably from the wrong version..
If you are running an new os run /Users/onlineforce/Desktop/NVIDIA-patcher.command md5 and ask the maintainer of this script to add support for your system
